src/index.ts

```diff
@@ -50,9 +58,20 @@ async function run() {
         ).stdout.includes('filter: lfs');
 
         if (shouldBeStoredInLFS) {
-          const isStoredInLFS = Boolean(
-            file.patch?.includes('version https://git-lfs.github.com/spec/v1')
-          );
+          let isStoredInLFS = false;
+          if (file.patch) {
+            isStoredInLFS = Boolean(
+              file.patch.includes('version https://git-lfs.github.com/spec/v1')
```

Comment:
nit: should this re-use the constant `lfsFileContentStart` we have at the top ? 

```suggestion
              file.patch.includes(lfsFileContentStart)
```

Reply:
oops, yeap!

done in 37d29229f497b4d749be453a83f2dda1bd222d64